README.md

```diff
@@ -3,17 +3,17 @@
 This repository contains a suite of tools that can be used to manage key areas
 of your GCP organization.
 
-- Deploy
-- Monitor
-- Audit
+-   Deploy
```

Comment:
I mentioned this in Martin's PR too but this is actually in violation of markdown rule https://github.com/DavidAnson/markdownlint/blob/master/doc/Rules.md#md030. Please use https://github.com/DavidAnson/vscode-markdownlint or https://github.com/igorshubovych/markdownlint-cli

Reply:
Done.